micro-airtable-api.test.js

```diff
@@ -0,0 +1,77 @@
+const http = require('http');
+const got = require('got');
+const listen = require('test-listen');
+const handler = require('./index');
+
+const describeIfEnv = (process.env.AIRTABLE_API_KEY && process.env.AIRTABLE_BASE_ID) ? describe : xdescribe;
+const baseConfig = {
+  airtableBaseId: process.env.AIRTABLE_BASE_ID,
+  airtableApiKey: process.env.AIRTABLE_API_KEY,
+};
+
+const getExistingPosts = async (client) => {
+  return (await client.get('/v0/Posts', {json: true})).body;
+};
+
+describeIfEnv('micro-airtable-api', () => {
+
+  describe('when `allowedMethods="*"`', () => {
+    const config = {...baseConfig, allowedMethods: '*'};
+    let server;
+    let client;
+
+    beforeAll(async () => {
```

Comment:
I just tried the test out locally and got the following error:

```
Jest did not exit one second after the test run has completed.

This usually means that there are asynchronous operations that weren't stopped in your tests. Consider running Jest with `--detectOpenHandles` to troubleshoot this issue.
```

We may need to close out the server after the test are done like so:

```js
beforeAll(async () => {
  // creating server...
});

afterAll(() => {
  server.close();
});
```

Adding this fixed the warning message for me.

Reply:
Interesting. I have not seen that error and I've run it dozens of times locally, but I added the `afterAll` call just to be sure. Thanks!